src/main.rs

```diff
@@ -758,25 +763,35 @@ fn handle_window_event<T>(
                             game.is_fullscreen = !game.is_fullscreen;
                         }
                         (ElementState::Pressed, Some(key)) => {
-                            if let Some(action_key) =
+                            /*if game.server.is_some() {
+                                if let Some(action_key) =
                                 settings::Actionkey::get_by_keycode(key, &game.vars)
-                            {
-                                if game.server.is_some() {
+                                {
                                     game.server.as_ref().unwrap().key_press(
                                         true,
                                         action_key,
                                         &mut game.screen_sys,
                                         &mut game.focused,
                                     );
                                 }
-                            }
+                            }*/
                             if !game.focused {
                                 let ctrl_pressed = game.is_ctrl_pressed || game.is_logo_pressed;
                                 ui_container.key_press(game, key, true, ctrl_pressed);
                             }
+                            if game.screen_sys.clone().press_key(key, true, game) {
```

Comment:
why are we cloning if we are checking something?

Reply:
Idk, i just added clones everywhere, i'll remove the redundant ones later

Reply:
Clippy will complain about it anyway, no need to review such things if the PR is not final